src/components/Transaction.vue

```diff
@@ -167,7 +167,8 @@ export default {
   computed: {
     address() {
       if (this.type === 'POSITIVE') {
-        return this.inputs.length > 1
+        return this.inputs.length > 1 &&
+          this.inputs.reduce((acc, input) => input.address !== acc.address)
```

Comment:
Here you want to check if there is at least one address different from the first one. You don't need reduce:

```suggestion
          this.inputs.some(input => input.address !== first_address)
```